ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json

```diff
@@ -136,5 +136,11 @@
     "CaseEventID": "CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_SETTLED_FOR_DEFENDANT1",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_DASHBOARD_NOTIFICATION_CP_TRIAL_ARRANGEMENTS_BOTH_PARTIES",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRU"
```

Comment:
Either this authorisation needs to be 2, or the events below need to be one - I think something was missed out.

Reply:
I have push the wrong file :) thanks max